test-framework/vault/src/main/java/io/quarkus/vault/runtime/client/dto/VaultSealStatus.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.vault.runtime.client.dto;
+
+import javax.json.bind.annotation.JsonbProperty;
+
+/**
+ * {
+ * "type":"shamir",
+ * "initialized":true,
+ * "sealed":false,
+ * "t":1,
+ * "n":1,
+ * "progress":0,
+ * "nonce":"",
+ * "version":"1.2.2",
+ * "migration":false,
+ * "cluster_name":"vault-cluster-21617f37",
+ * "cluster_id":"c47a6792-ec86-8e59-0747-5729f95db971",
+ * "recovery_seal":false
+ * }
+ */
+public class VaultSealStatus implements VaultModel {
```

Comment:
Is it normal that these ones are in the test framework?

If so, we need to adjust the package as we have split packages right now.

Reply:
@sberyozkin insisted that only the operations that were actually used at runtime would be present on the client interface. those extra methods are used only in the context of the tests, such as initializing the vault. my preference was to have them on the real client interface, and just use them from the tests.
I do not mind putting all methods back onto VaultClient (which would have the benefit of dropping TestVaultClient as well), if you want this and sergey is ok with it.

in the meantime I am just going to move all quarkus-vault-test classes to io.quarkus.vault.test.*

Reply:
@vsevel @gsmet yes, I was keeping nudging Vincent  :-) to remove it completely to minimize a number of classes in the main/src given the PR was large, and in the end we ended up with a compromise to keep those extra classes in the tests only for Vincent's tests work against a complete Vault API; @gsmet, if you also prefer to move those classes back to the main source then I'd be fine with it, thanks

Reply:
No, I'm OK with it. It's just that we don't want split packages so changing the package name is sufficient.